draft/webxdc-dev-reference.md

```diff
@@ -54,7 +54,7 @@ and you won't get the update by `setUpdateListener()`.
 ### setUpdateListener()
 
 ```js
-window.webxdc.setUpdateListener((update) => {}, serial) -> Promise<void>;
+promise = window.webxdc.setUpdateListener((update) => {}, serial);
```

Comment:
please add `let` or `const` before that (`var` is also still better than nothing)